.github/workflows/replay-verify-testnet.yaml

```diff
@@ -17,6 +17,7 @@ env:
   BUCKET: aptos-testnet-backup-2223d95b
   SUB_DIR: e1
   HISTORY_START: 350000000
```

Comment:
Yep. I intentionally set this because I don't yet know what are the other versions that broke the replay. I'm planning to run it manually to see the configs for testnet.